jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java

```diff
@@ -69,20 +68,24 @@
 
     private static final int WHEEL_SCALE = 120;
 
-    private LwjglWindow context;
+    private final LwjglWindow context;
     private RawInputListener listener;
     private boolean cursorVisible = true;
+    private Long[] currentCursor;
```

Comment:
Why do you use Long[] instead of long[]?

Reply:
Because I use it in the Map as well :(

Reply:
you can use long[] in all places.

Reply:
Hmm, yesh, it is an object. Should go in there. Instinctively added value type because Map...